ledger/participant-integration-api/src/test/suite/scala/platform/apiserver/execution/StoreBackedCommandExecutorSpec.scala

```diff
@@ -4,13 +4,12 @@
 package com.daml.platform.apiserver.execution
 
 import java.time.Duration
-
```

Comment:
./fmt.sh changes

Reply:
I'm confused, CI checks formatting so `./fmt.sh` should never change anything without you making a change to that file Are you sure you did not run intellij formatting instead?